tests/Bundle/CoverallsBundle/Console/ApplicationTest.php

```diff
@@ -21,7 +20,12 @@ public function shouldExecuteCoverallsJobsCommand()
         $this->makeProjectDir(null, $this->logsDir);
         $this->dumpCloverXml();
 
-        $app = new Application($this->rootDir, 'Coveralls API client for PHP', '1.0.0');
+        if (PHP_VERSION_ID >= 71000) {
```

Comment:
It's not about PHP version but about installed Symfony version

Reply:
Yes, now I totally changed conception